### PR TITLE
BZ1899171

### PR DESCRIPTION
**Pull request by stevsmit**
For 4.6.

BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1899171

Preview at https://deploy-preview-30735--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-6-release-notes.html#ocp-4-6-networking

It has been requested that I submit a change notification. 

@sferich888 @xltian @sjstout @Miciah @mcurry-rh Please review and let me know if further changes are needed.

### CODE REVIEW

**Comment by sjstout** [comment]
@stevsmit , LGTM! I also double-checked the Tech Preview table in the 4.6 Release Notes, and they correctly reflect the GA status for OVN. Thanks!

**Comment by stevsmit** [comment]
Retagging @sferich888  @xltian  @Miciah  @mcurry-rh in hopes to get this merged by the end of the week. Thanks!

File: release_notes/ocp-4-6-release-notes.adoc [hunk 1]
```diff
@@ -709,6 +709,13 @@ AWS Route53 endpoint configuration is now supported on the Ingress Operator.
 
 See xref:../installing/installing_aws/installing-aws-account.html#nw-endpoint-route53_installing-aws-account[Ingress Operator endpoint configuration for AWS Route53] for more information.
 
+[id="ocp-4-6-ingress-controller-unique-id-header"]
+==== Ingress Controller unique-id configuration
+
+Configuring an Ingress Controller to inject an HTTP header with a uniquely defined request ID is now supported. This can be used to trace cluster traffic.
```
Comment (sferich888) [inline release_notes/ocp-4-6-release-notes.adoc:715]: Lets denote the z stream / errata that provided this functionality with 4.6

**Comment by stevsmit** [comment]
Approved by Marc Curry on 4/7.

File: release_notes/ocp-4-6-release-notes.adoc (hunk 1 above)
Comment (stevsmit) [reply release_notes/ocp-4-6-release-notes.adoc:715]: Ingress specification and uniqueID are in 4.7+ (https://docs.openshift.com/container-platform/4.7/rest_api/operator_apis/ingresscontroller-operator-openshift-io-v1.html#specification).

**Comment by xltian** [comment]
@lihongan to review from QE

**Comment by lihongan** [comment]
@quarterpin Could you please help take a look? thanks.

File: release_notes/ocp-4-6-release-notes.adoc [hunk 2]
```diff
@@ -709,6 +709,13 @@ AWS Route53 endpoint configuration is now supported on the Ingress Operator.
...
+
+For more information, see xref:../rest_api/operator_apis/ingresscontroller-operator-openshift-io-v1.adoc#specification[IngressController Specification]
```
Comment (bergerhoffer) [inline release_notes/ocp-4-6-release-notes.adoc:717]: * Missing a period.
* I'd put "IngressController" in backticks, and lowercase "specification". And probably a "the " in front of the link.

Comment (stevsmit) [reply release_notes/ocp-4-6-release-notes.adoc]: Fixed!

**Comment by bergerhoffer** [comment]
LGTM now!

**Pull request merged**